apps/condo/domains/common/components/TicketCard/TicketCardList.tsx

```diff
@@ -0,0 +1,169 @@
+    const query = qs.stringify(
+        {
+            filters: JSON.stringify(pickBy(filters)),
+        },
+        { arrayFormat: 'comma', skipNulls: true, addQueryPrefix: true },
+    )
+
+    return (
+        <Container >
+            <AddressPartContainer>
+                <Space size={8} direction={'vertical'}>
+                    <Typography.Text type='secondary'>
+                        {AddressLabel}
+                    </Typography.Text>
+                    <Typography.Title level={4}>
+                        {addressPrefix}
+                    </Typography.Title>
+                </Space>
+            </AddressPartContainer>
+            <TicketsPartContainer>
+                <Row gutter={TICKET_CARD_GUTTER}>
+                    <Col span={24}>
+                        <>
+                            {tickets.length > 0
+                                ?
+                                <Typography.Title level={5} >
+                                    {TicketsByContactMessage}
+                                </Typography.Title>
+                                :
+                                <Typography.Text>
+                                    {NoTicketsOnAddressMessage}
+                                </Typography.Text>
+                            }
+                            {
+                                tickets.slice(0, TICKETS_ON_CARD).map((ticket) => {
+                                    return <TicketOverview
+                                        key={ticket.id}
+                                        id={ticket.id}
+                                        details={ticket.details}
+                                        createdAt={ticket.createdAt}
+                                        number={ticket.number}
+                                        status={ticket.status.name}/>
+                                })
+                            }
+                            {hasMoreTickets > 0 && (
+                                <Col span={24}>
+                                    <Link href={`/ticket/${query}`}>
+                                        <Typography.Link style={TICKET_CARD_HAS_MORE_LINK_STYLE}>
+                                            {intl.formatMessage({ id: 'MoreTicketsLeft' }, {
+                                                ticketsLeft: hasMoreTickets,
+                                            })}
+                                        </Typography.Link>
+                                    </Link>
+                                </Col>
+                            )}
+                        </>
+                    </Col>
+                </Row>
+            </TicketsPartContainer>
+        </Container>
+    )
+}
+
+const TicketCardList: React.FC<ITicketCardListProps> = ({ organizationId, contactPhone, contactName }) => {
+    const {
+        loading,
+        objs: tickets,
+    } = Ticket.useObjects({
+        sortBy: ['createdAt_DESC'] as SortTicketsBy[],
```

Comment:
`sortBy` value looks like a constant, non changing structure

Reply:
fixed this